source/composeWith.js

```diff
@@ -0,0 +1,31 @@
+import _curry2 from './internal/_curry2';
+import pipeWith from './pipeWith';
+import reverse from './reverse';
+
+
+/**
+ * Performs right-to-left function composition using transforming function. The rightmost function may have
+ * any arity; the remaining functions must be unary.
+ *
+ * **Note:** The result of compose is not automatically curried.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (((a, b, ..., n) -> o), [(y -> z), (x -> y), ..., (o -> p)]) -> ((a, b, ..., n) -> z)
```

Comment:
This signature isn't right. Sorry I didn't catch it the first time. The first function is interpolated between all the others, so it won't take the initial parameters. For lack of a better option, I think I'd be OK with `* -> *`